initialization/keygen/README.md

```diff
@@ -5,6 +5,7 @@
 Generate keypairs for use in other applications. The repository contains generators for the following cryptosystems:
 - ed25519
 - secp256k1
+- x25519
```

Comment:
should we add single line descriptions? Maybe link to some guide that explains all three.

Reply:
yeah, added